### PR TITLE
Ft next project init

### LINKED ISSUE #766: Fix next platform deploy previews

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Adds boilerplate code for the next platform
- [x] #758
- [x] #778
- [x] #779
- [x] Closes #766 

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [x] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please check the corresponding PRs mentioned in this PR summary for how to test the individual changes.

#### What are the relevant tickets?
- [OPS-28](https://airqoteam.atlassian.net/browse/OPS-28?atlOrigin=eyJpIjoiOWMzNTRhZmU4ZjAxNDA4NjgyYTNlOGNmNThlOGJhNDkiLCJwIjoiaiJ9)

#### Screenshots (optional)
image
image
image

### CODE REVIEW

**Comment by 123MwanjeMike** [comment]
Advanced Cypress e2e test case examples can always be found [here](https://github.com/airqo-platform/AirQo-frontend/tree/eee8cff3b64e621797fd1bfe37445fffec134cb5/platform/cypress/e2e/2-advanced-examples) cc @Codebmk @FaithDaka

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Review by Baalmart — COMMENTED** [review]
Thanks @123MwanjeMike , this LGTM.

A few clarifications:

1. Does this project have a README with run instructions for some of these recent configurations?
2. Is it possible to "temporarily disable" the **dependebots** for this project using this PR or that is another issue?

Thank you!

**Comment by 123MwanjeMike** [comment]
> Thanks @123MwanjeMike , this LGTM.
> 
> A few clarifications:
> 
>     1. Does this project have a README with run instructions for some of these recent configurations?
> 
>     2. Is it possible to "temporarily disable" the **dependebots** for this project using this PR or that is another issue?
> 
> 
> Thank you!

Done

**Review by Codebmk — APPROVED** [review]
The setup is running great for me. Thanks Mike

**Review by Mnoble-19 — APPROVED** [review]
LGTM

File: .github/workflows/deploy-frontends-to-staging.yml [hunk 1]
```diff
@@ -138,7 +145,7 @@ jobs:
           docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/airqo-stage-platform-frontend:${{ needs.image-tag.outputs.build_id }}
       - name: Deploy to K8S
         run: |
-          kubectl apply -f k8s/platform/stage-airqo-platform-frontend.yaml
+          kubectl apply -f k8s/netmanager/stage-airqo-platform-frontend.yaml
```
Comment (Baalmart) [inline .github/workflows/deploy-frontends-to-staging.yml:148]: Hi @123MwanjeMike , thanks for this amazing work. I sense that things are taking great shape. Brilliant file changes! :) 

Please shed more light on this change in name from platform to netmanager? In summary, what does it mean in the current context where we shall two platform projects running at the same time?

**Review by Baalmart — COMMENTED** [review]
Thanks @123MwanjeMike , just left one review comment in one of the changed files. Thanks again!

File: .github/workflows/deploy-frontends-to-staging.yml (hunk 1 above)
Comment (123MwanjeMike) [reply .github/workflows/deploy-frontends-to-staging.yml:148]: The old one(netmanager) will be phased out as soon as the new one is available. However, this changes is to allow maintenance works reflect on the old one.

**Review by Baalmart — APPROVED** [review]
LGTM

**Review by FaithDaka — APPROVED** [review]

**Pull request merged**